application-licensing-licensor/application-licensing-licensor-ui/src/main/resources/Licenses/Code/MissingLicensesUIX.xml

```diff
@@ -120,6 +223,25 @@
     !$xwiki.exists('XWiki.Alerts.Code.AlertsMenuUIX'))
   {{include reference="Licenses.Code.LicensesNotificationsUIX" /}}
 #end
+
+#set ($unlicensedSheets = $services.licensing.licensor.getUnlicensedSheets($doc.documentReference, $xcontext.action))
+#if ($services.sheet.getSheets($doc, $xcontext.action).isEmpty() &amp;&amp; $unlicensedSheets.size() &gt; 0)
+  #set ($extensions = $collectionstool.set)
+  #foreach ($sheet in $unlicensedSheets)
+    #foreach ($extension in $services.extension.xar.getInstalledExtensions($sheet))
+      #set ($discard = $extensions.add($extension.name))
+    #end
+  #end
+  {{html clean="false"}}
+  ## Create a div that will display a warning and the applications that don't have a license.
+  ## Because the "After container header" UIX (org.xwiki.platform.template.content.header.after) is available since
+  ## XWiki 10.4RC1 and we need to support XWiki 8.4+, the div will be created and with hidden class. After that, it will bemoved with JS.
```

Comment:
Line too long (>121 chars) and there are some typos: "will be created and with" and "bemoved". You should review your changes before committing.